ui/src/main/java/edu/wpi/grip/ui/OperationListController.java

```diff
@@ -0,0 +1,105 @@
+package edu.wpi.grip.ui;
+
+import com.google.common.annotations.VisibleForTesting;
+import com.google.common.eventbus.Subscribe;
+import com.sun.javafx.application.PlatformImpl;
+import edu.wpi.grip.core.Operation;
+import edu.wpi.grip.core.events.OperationAddedEvent;
+import edu.wpi.grip.ui.annotations.ParametrizedController;
+import edu.wpi.grip.ui.util.ControllerMap;
+import edu.wpi.grip.ui.util.SearchUtility;
+import javafx.beans.InvalidationListener;
+import javafx.beans.property.SimpleStringProperty;
+import javafx.beans.property.StringProperty;
+import javafx.collections.MapChangeListener;
+import javafx.fxml.FXML;
+import javafx.scene.Node;
+import javafx.scene.control.Tab;
+import javafx.scene.layout.VBox;
+
+import javax.inject.Inject;
+
+/**
+ * Controller for a VBox of {@link OperationController}s.  The user data for this should be what category it shows,
+ * and the filterText property allows for fuzzy searching the operations.
+ *
+ * @see edu.wpi.grip.core.Operation.Category
+ */
+@ParametrizedController(url = "OperationList.fxml")
+public class OperationListController {
+
+    public final static String FILTER_TEXT = "filterText";
+
+    @FXML private Tab root;
+    @FXML private VBox operations;
+
+    @Inject private OperationController.Factory operationControllerFactory;
+
+    private final StringProperty filterText = new SimpleStringProperty(this, FILTER_TEXT, "");
+    private String baseText = null;
```

Comment:
No, I already told you  **that doesn't work**.  Did you even try running that code?  It keeps getting the current text in the title and setting it to `baseText`.  `baseText` is supposed to have just the title text without the parentheses and number.  Retrieving it over and over again means you'll have a bunch of those at the end of the title (like "Image processing (1) (2) (3) (4) (5)...".